### DISCUSSION TITLE
2.1.3

### DISCUSSION THREAD

**Discussion by T8RIN** [T8RIN/ImageToolbox#286] [Announcements]
## What's Changed
(Mostly fixed all of existing issues of 2.1.1, so changes are the same)
* ### App now named "Image Toolbox" in favour of better understanding it main functions, because now we not just resize images, we can almost everything
* Added ability to apply filter chains to images and also 50 filters included on the start
* Resize by limits option added
* Loading image from internet - now you can easily download some images from web
* Fix previewing large images
* Added option to compare output and start image
* Now you can hold and get tooltips on each resize type: Explicit, Flixible, Ratio
* Shared images preview like in android 12 sheet
* Bumped dependencies
* Fixed critical bugs
* Updated UI
* Added new emojis

Thanks for using Image Toolbox (Image Resizer) stay tuned for the next updates ❤️ 

**Full Changelog**: https://github.com/T8RIN/ImageToolbox/compare/2.1.1...2.1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/T8RIN/ImageToolbox/releases/tag/2.1.3'>2.1.3</a>.</em>

**Comment by Shakeuptech** [comment]
Thank you for this wonderful app. 🙏🏻

**Reply by T8RIN** [reply]
Thanks for using it :)

**Comment by T8RIN** [comment]
The next version will contain the sketchbook function, i.e. drawing on image.

And one more thing, i plan to add something really cool (if it work as i expected), stay tuned!
